### PR TITLE
replaced old names AIBricks with AIBrix

### PR DESCRIPTION
**Pull request by nwangfw**
Replaced all old project names from AIBricks to AIBrix

### CODE REVIEW

File: config/samples/orchestration_v1alpha1_rayclusterfleet.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      models.aibricks.ai: llama-2-7b-hf
```
Comment (Jeffwan) [inline config/samples/orchestration_v1alpha1_rayclusterfleet.yaml:12]: I forgot to mention that, please update "models.aibricks.ai" -> "model.aibrix.ai/name"

File: config/samples/orchestration_v1alpha1_rayclusterfleet.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
   template:
     metadata:
       labels:
-        models.aibricks.ai: llama-2-7b-hf
+        models.aibrix.ai/name: llama-2-7b-hf
```
Comment (Jeffwan) [inline config/samples/orchestration_v1alpha1_rayclusterfleet.yaml]: it should be `model` instead of `models`, please help change all the references.

Comment (Jeffwan) [reply config/samples/orchestration_v1alpha1_rayclusterfleet.yaml]: @nwangfw

Comment (nwangfw) [reply config/samples/orchestration_v1alpha1_rayclusterfleet.yaml]: Thanks for your carefully check. I previously didn't notice this part.

**Pull request merged**